masterlist.yaml

```
@@ -21262,13 +21262,32 @@ plugins:
         nav: 1
 
   - name: 'RedBag''s Solitude.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/42052/' ]
-    dirty:
-      - <<: *quickClean
-        crc: 0xC025F482
-        util: '[SSEEdit v4.0.3g](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
-        itm: 95
-        udr: 79
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/42052/'
+        name: 'Redbag''s Solitude'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64068/'
+        name: 'Rob''s Bug Fixes - RedBag''s Solitude'
+    msg:
+      - <<: *patchUpdateAvailable
+        subs: [ '[Rob''s Bug Fixes - RedBag''s Solitude](https://www.nexusmods.com/skyrimspecialedition/mods/64068/)' ]
+        condition: 'checksum("RedBag''s Solitude.esp", A9A23AB8)'
+      - <<: *patch3rdParty
+        subs:
+          - 'Skyrim Project Optimization'
+          - '[Rob''s Bug Fixes - RedBag''s Solitude](https://www.nexusmods.com/skyrimspecialedition/mods/64068/)'
+        condition: 'active("Skyrim Project Optimization - Full ESL Version.esm") and not active("RedBag''s Solitude - SPO Patch.esp")'
```

Looking at the patch in xEdit, it's only necessary for your replacer plugin, so `patchIncluded` would actually be more appropriate. The messages should also be sorted alphabetically by alias:

```suggestion
      - <<: *patchIncluded
        subs: [ 'Skyrim Project Optimization' ]
        condition: 'active("Skyrim Project Optimization - Full ESL Version.esm") and checksum("RedBag''s Solitude.esp", 90AD69F2) and not active("RedBag''s Solitude - SPO Patch.esp")'
      - <<: *patchUpdateAvailable
        subs: [ '[Rob''s Bug Fixes - RedBag''s Solitude](https://www.nexusmods.com/skyrimspecialedition/mods/64068/)' ]
        condition: 'checksum("RedBag''s Solitude.esp", A9A23AB8)'
```